src/examples/forms/inputs/AutoExpand.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Textarea } from '@zendeskgarden/react-forms';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={5}>
+      <Field>
+        <Label>Ivy</Label>
+        <Textarea minRows={1} maxRows={12} />
```

Comment:
Since textareas normally include multi-line text we set the `[minRows=3]` in the products. This shows that difference in input fields when there is no text included. 

Should we do that here as well?